### ISSUE TITLE
Examples fail on pull from main branch

### ISSUE THREAD

**Issue by ronald-jaepel** [fau-advanced-separations/CADET-Process#27]
Hey,

I'm getting the following error when running the examples (both the "column_transport_parameters.py" and "particle_porosity.py" in the characterize_chromatographic_system folder: 

```There was an exception in simulate_n_cycles
Traceback (most recent call last):
  File "C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\log.py", line 172, in wrapper
    return function(*args, **kwargs)
  File "C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\simulator\simulator.py", line 253, in simulate_n_cycles
    return self.run(process, **kwargs)
  File "C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\simulator\cadetAdapter.py", line 259, in run
    cadet.root = self.get_process_config(process)
  File "C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\simulator\cadetAdapter.py", line 342, in get_process_config
    config.input.model = self.get_input_model(process)
  File "C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\simulator\cadetAdapter.py", line 611, in get_input_model
    input_model.connections = self.get_model_connections(process)
  File "C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\simulator\cadetAdapter.py", line 633, in get_model_connections
    section_states = process.flow_rate_section_states
  File "C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\dataStructure\cache.py", line 13, in __get__
    value = super().__get__(instance, cls)
  File "C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\processModel\process.py", line 215, in flow_rate_section_states
    for unit, unit_flow_rates in self.flow_rate_timelines.items():
  File "C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\dataStructure\cache.py", line 13, in __get__
    value = super().__get__(instance, cls)
  File "C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\processModel\process.py", line 161, in flow_rate_timelines
    flow_rates = self.flow_sheet.get_flow_rates(state)
  File "C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\processModel\flowSheet.py", line 591, in get_flow_rates
    solution = self.solve_flow_rates(flow_rates, output_states, i)
  File "C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\processModel\flowSheet.py", line 733, in solve_flow_rates
    solution = sym.solve(unit_total_flow_eq + unit_outflow_eq, symbols)
  File "C:\Users\ronal\miniconda3\envs\jupyterlab\lib\site-packages\sympy\solvers\solvers.py", line 908, in solve
    raise ValueError('duplicate symbols given')
ValueError: duplicate symbols given

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[7], line 5
      2 simulator = Cadet()
      4 if __name__ == '__main__':
----> 5     simulation_results = simulator.simulate(process)
      6     _ = simulation_results.solution.outlet.inlet.plot()

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\simulator\simulator.py:199, in SimulatorBase.simulate(self, process, previous_results, **kwargs)
    197 process.lock = True
    198 if not self.evaluate_stationarity:
--> 199     results = self.simulate_n_cycles(
    200         process, self.n_cycles, previous_results, **kwargs
    201     )
    202 else:
    203     results = self.simulate_to_stationarity(
    204         process, previous_results, **kwargs
    205     )

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\log.py:150, in log_time.<locals>.log_time_decorator.<locals>.wrapper(*args, **kwargs)
    147 @wraps(function)
    148 def wrapper(*args, **kwargs):
    149     start = time.time()
--> 150     result = function(*args, **kwargs)
    151     elapsed = time.time() - start
    152     logger = get_logger(logger_name, level=None)

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\log.py:201, in log_results.<locals>.log_results_decorator.<locals>.wrapper(*args, **kwargs)
    197 logger = get_logger(logger_name, level=None)
    199 logger.debug('{} was called with {}, {}'.format(
    200         function, *args, **kwargs))
--> 201 results = function(*args, **kwargs)
    202 logger.debug(f'Results: {results}')
    204 return results

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\log.py:180, in log_exceptions.<locals>.log_exception_decorator.<locals>.wrapper(*args, **kwargs)
    177 logger.exception(err)
    179 # re-raise the exception
--> 180 raise e

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\log.py:172, in log_exceptions.<locals>.log_exception_decorator.<locals>.wrapper(*args, **kwargs)
    170 logger = get_logger(logger_name, level=None)
    171 try:
--> 172     return function(*args, **kwargs)
    173 except Exception as e:
    174     # log the exception
    175     err = "There was an exception in "

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\simulator\simulator.py:253, in SimulatorBase.simulate_n_cycles(self, process, n_cyc, previous_results, **kwargs)
    250 if previous_results is not None:
    251     self.set_state_from_results(process, previous_results)
--> 253 return self.run(process, **kwargs)
    255 self.n_cycles = n_cyc_orig

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\simulator\cadetAdapter.py:259, in Cadet.run(self, process, cadet, file_path)
    256 if cadet is None:
    257     cadet = CadetAPI()
--> 259 cadet.root = self.get_process_config(process)
    261 if cadet.is_file:
    262     if file_path is None:

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\simulator\cadetAdapter.py:342, in Cadet.get_process_config(self, process)
    339 process.lock = True
    341 config = Dict()
--> 342 config.input.model = self.get_input_model(process)
    343 config.input.solver = self.get_input_solver(process)
    344 config.input['return'] = self.get_input_return(process)

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\simulator\cadetAdapter.py:611, in Cadet.get_input_model(self, process)
    595 """Config branch /input/model/
    596 
    597 Notes
   (...)
    607 
    608 """
    609 input_model = Dict()
--> 611 input_model.connections = self.get_model_connections(process)
    612 # input_model.external = self.model_external # !!! not working yet
    613 input_model.solver = self.model_solver_parameters.to_dict()

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\simulator\cadetAdapter.py:633, in Cadet.get_model_connections(self, process)
    629     model_connections['CONNECTIONS_INCLUDE_DYNAMIC_FLOW'] = 1
    631 index = 0
--> 633 section_states = process.flow_rate_section_states
    635 for cycle in range(0, self.n_cycles):
    636     for flow_rates_state in section_states.values():

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\dataStructure\cache.py:13, in cached_property_if_locked.__get__(self, instance, cls)
     10     except KeyError:
     11         pass
---> 13 value = super().__get__(instance, cls)
     15 if instance.lock:
     16     instance.cached_properties[self.name] = value

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\processModel\process.py:215, in Process.flow_rate_section_states(self)
    203 section_states = {
    204     time: {
    205         unit.name: {
   (...)
    211     } for time in self.section_times[0:-1]
    212 }
    214 for sec_time in self.section_times[0:-1]:
--> 215     for unit, unit_flow_rates in self.flow_rate_timelines.items():
    216         if isinstance(self.flow_sheet[unit], Inlet):
    217             section_states[sec_time][unit]['total_in'] \
    218                 = unit_flow_rates['total_out'].coefficients(sec_time)[0]

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\dataStructure\cache.py:13, in cached_property_if_locked.__get__(self, instance, cls)
     10     except KeyError:
     11         pass
---> 13 value = super().__get__(instance, cls)
     15 if instance.lock:
     16     instance.cached_properties[self.name] = value

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\processModel\process.py:161, in Process.flow_rate_timelines(self)
    158 start = self.section_times[i]
    159 end = self.section_times[i+1]
--> 161 flow_rates = self.flow_sheet.get_flow_rates(state)
    163 for unit, flow_rate in flow_rates.items():
    164     unit_flow_rates = flow_rate_timelines[unit]

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\processModel\flowSheet.py:591, in FlowSheet.get_flow_rates(self, state)
    586 origin_flow_rates = {
    587     unit.name: defaultdict(list_factory) for unit in self.units
    588 }
    590 for i in range(4):
--> 591     solution = self.solve_flow_rates(flow_rates, output_states, i)
    592     if solution is not None:
    593         for unit_index, unit in enumerate(self.units):

File C:\Users/ronal/Documents/CADET-Process-master/CADET-Process-master\CADETProcess\processModel\flowSheet.py:733, in FlowSheet.solve_flow_rates(self, inlet_flow_rates, output_states, coeff)
    726 # Solve system of equations
    727 symbols = (
    728     *unit_total_flow_symbols,
    729     *unit_inflow_symbols,
    730     *unit_outflow_symbols
    731 )
--> 733 solution = sym.solve(unit_total_flow_eq + unit_outflow_eq, symbols)
    735 solution = {str(key): value for key, value in solution.items()}
    737 return solution

File ~\miniconda3\envs\jupyterlab\lib\site-packages\sympy\solvers\solvers.py:908, in solve(f, *symbols, **flags)
    906 # check for duplicates
    907 if len(symbols) != len(set(symbols)):
--> 908     raise ValueError('duplicate symbols given')
    909 # remove those not of interest
    910 exclude = flags.pop('exclude', set())

ValueError: duplicate symbols given```

Do you recognize this error?

**Comment by ronald-jaepel** [comment]
Code is from commit [d977d7](https://github.com/fau-advanced-separations/CADET-Process/commit/d977d79cd3364711e4f3e00995f8af8f1b54716b)

**Comment by schmoelder** [comment]
This is probably because of an update of sympy.

Can you check

```
pip show sympy
```

If it's 1.12 or higher, please downgrade to 1.11

I believe that this issue should already have been fixed with 94aca9a4fd88951128909b1a47a6f6a0622571ca. Your code appears to be an older version, however, so please try upgrading again.

https://github.com/fau-advanced-separations/CADET-Process/blob/94aca9a4fd88951128909b1a47a6f6a0622571ca/CADETProcess/processModel/flowSheet.py#L732
 
Regardless, there is still an issue with Polyround (or one of its dependencies) which prevents us from bumping the version (see https://gitlab.com/csb.ethz/PolyRound/-/issues/6).

**Comment by ronald-jaepel** [comment]
Solved
